curp/src/server/mod.rs

```diff
@@ -557,9 +564,7 @@ impl<C: 'static + Command> Protocol<C> {
         if req.term > state.term {
             state.update_to_term(req.term);
```

Comment:
`update_to_term` will  change `leader_id` to `None`, other places where called this function may not immediately update the `leader_id`, e.g.: [link](https://github.com/themanforfree/Xline/blob/780f5bbc537d786b71cfe0e143448938cf49adea/curp/src/server/bg_tasks.rs#L195), so there will be a period of time we don't know who the leader is, should we notify this case to xline?

Reply:
Fixed